src/main/java/com/employeeManagement/employee/Employee.java

```diff
@@ -10,7 +10,6 @@
 
 @Entity
 public class Employee {
-    

```

Comment:
i need this empty line please don't delete it

Reply:
ok